client/transaction-builder/src/stdlib.rs

```diff
@@ -3096,8 +3096,8 @@ impl ScriptCall {
 }
 
 impl ScriptFunctionCall {
-    /// Build a Diem `TransactionPayload` from a structured object `ScriptFunctionCall`.
-    pub fn encode(self) -> TransactionPayload {
+    /// Build a Diem `ScriptFunction` from a structured object `ScriptFunctionCall`.
+    pub fn encode(self) -> ScriptFunction {
```

Comment:
I think it makes it extremely awkward for any use-site I had. Since you had to manually unwrap it back into a `ScriptFunction`. Can't these other use sites just wrap it with a `TransactionPayload`?

Reply:
Agreed!